cloudflare-d1/.gitignore

```diff
@@ -0,0 +1,21 @@
+.env
+!.env.example
+.DS_Store
+.react-router
+build
+node_modules
+*.tsbuildinfo
+
+# Cloudflare specific
+.mf
+.wrangler
```

Comment:
I think we just want this portion, the netlify/vercel stuff shouldn't be in here

Also, I just merged #4, which covered the missing netlify `.gitignore`